### DISCUSSION TITLE
Wildcard filtering for iteration field

### DISCUSSION THREAD

**Discussion by pat-sc-philips** [community/community#118698] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Hi,

I would like to use a wildcard to filter my project view when I'm using an "iteration " field. If I'm not mistaken, wildcard is supported but only for range.

My use case: I have the following iteration field called Sprint with the following value "Sprint 2024.1", "Sprint 2024.2", "Sprint 2024.3", "Sprint 2024.4", "Sprint 2025.1".

If I want to have all 2024 sprints displayed, I can do the following: `sprint:"Sprint 2024.1..Sprint 2024.4"`

What I would like to be able to do is the following: `sprint:"Sprint 2024.*"`

Thank you

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by joshjohanning** [comment]
Hmm, I could not get this to work with the Iteration field, but I could get it to work with the Milestone field. 
![image](https://github.com/community/community/assets/19912012/992ce876-bebd-4631-98ff-699d5b4f5892)

Perhaps it is because Milestone is not a "custom field" - it's related to an issue - and iteration is a "custom project field" but just a special, date-based one. 

I know milestone doesn't provide the same functionality. I wonder if you could create some automation that when an issue gets labeled as "2024.1" it also assigns it to that milestone.  Or update this manually :/

**Comment by pat-sc-philips** [comment]
Thank for your reply, much appreciated.

Right now, I already have a workaround, I'm using the interval filter (`sprint:"Sprint 2024.1..Sprint 2024.4"`) so "I'm good".

However, I was sharing the feedback because

1. I feel the "expectation" for a wildcard is to work like in my initial attempt: `sprint:"Sprint 2024.*`
2. I guess if I got confused, I won't be the only one.

We are evaluating if we could transition from JIRA to GitHub projects. This is not a blocker, but a small annoyance.
